### DISCUSSION TITLE
Version string empty

### DISCUSSION THREAD

**Discussion by T3jv1l** [projectdiscovery/nuclei#2715] [Q&A] [answered]
Nuclei version 2.7.7 i think is latest

### Current Behavior:
I try to install nuclei template in my personal docker (docker image Ubuntu 20.04) for bug bounty and i encounter the issue with nuclei.

### Anything else:
![image](https://user-images.githubusercontent.com/33009059/193259040-ac550d1f-b4a1-4ec2-9313-bc29a2b13d77.png)

**Comment by T3jv1l** [comment]
I figure out this is happened because nuclei don't know how to pull the template in /root directory automatically.

**Comment by ehsandeep** [comment, marked as answer]
There are cases on the host/network where Cloudflare is blocked, as a result, version API is not reached and so the error.

**Comment by T3jv1l** [comment]
Thank you for all support!
